aspnetcore/security/authentication/community.md

```diff
@@ -17,9 +17,8 @@ The list below is sorted alphabetically.
 
 | Name | Description |
 | ---- | ----------- |
-| [AspNet.Security.OpenIdConnect.Server (ASOS)](https://github.com/aspnet-contrib/AspNet.Security.OpenIdConnect.Server) | ASOS is a low-level, protocol-first OpenID Connect server framework for ASP.NET Core and OWIN/Katana. |
 | [Gluu Server](https://gluu.org/) | Enterprise ready, open source software for identity, access management (IAM), and single sign-on (SSO). For more information, see the [Gluu Product Documentation](https://gluu.org/docs/). |
 | [IdentityServer](https://identityserver.io/) | IdentityServer is an OpenID Connect and OAuth 2.0 framework for ASP.NET Core, officially certified by the OpenID Foundation and under governance of the .NET Foundation. For more information, see [Welcome to IdentityServer4 (Documentation)](https://identityserver4.readthedocs.io/en/latest/). |
-| [OpenIddict](https://github.com/openiddict/openiddict-core) | OpenIddict is an easy-to-use OpenID Connect server for ASP.NET Core. |
+| [OpenIddict](https://github.com/openiddict/openiddict-core) | Versatile OpenID Connect stack for ASP.NET Core 2.1/3.1/5.0 and OWIN/Katana 4.1 (compatible with ASP.NET 4.6.1). |
```

Comment:
```suggestion
| [OpenIddict](https://github.com/openiddict/openiddict-core) | Versatile OpenID Connect stack for ASP.NET Core 2.1/3.1/5.0 and Microsoft.Owin 4.1 (compatible with ASP.NET 4.6.1). |
```

Reply:
Hum, why not. I’ll update the repo description to match that.

Reply:
It's just a branding thing. Microsoft.Owin is the closest thing to a product name. Katana is just a project code name.

Reply:
Yeah, makes sense (but Katana was such a cool name!)

Reply:
There was even an actual sword at one point 😁.

Reply:
No, really?! Do you have a picture of it? 😲